config.md

```diff
@@ -323,7 +328,7 @@ For Windows based systems the user structure has the following fields:
 ## <a name="configPlatformSpecificConfiguration" />Platform-specific configuration
 
 [**`platform.os`**](#platform) is used to specify platform-specific configuration.
-Runtime implementations MAY support any valid values for platform-specific fields as part of this configuration.
+Runtimes MAY support any valid values for platform-specific fields as part of this configuration.
```

Comment:
Fwiw, I think "Runtime implementations" is better than 'Runtimes" and I'd change the three 'Runtimes' below to that too.  My $.02, your call to make.